assets/onboarding/purpose/index.jsx

```diff
@@ -45,11 +46,17 @@ const purposes = [
 export const Purpose = () => {
 	const { goTo } = useQueryStringRouter();
 
+	const { data, submit, isBusy } = useOnboardingApi( 'purpose' );
+
 	const [ { selected, other }, setFormState ] = useState( {
 		selected: [],
 		other: '',
 	} );
 
+	useEffect( () => {
+		if ( data && data.selected ) setFormState( data );
```

Comment:
We should [always use braces and multiple lines](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#blocks-and-curly-braces) for conditionals.

Reply:
ab56f64061a85a7151b664eb1b305282d8d254de